src/main/resources/sparql/select-agent-summaries.rq

```diff
@@ -26,10 +26,15 @@ WHERE {
 	    	?currentDescription rdaa:P50032 ?label .
 	    }
 	    UNION {
-	        ?currentDescription rdaa:P50292 ?givenName  .
-	        ?currentDescription rdaa:P50291 ?surname  .
-            BIND(CONCAT(?givenName," ",?surname) as ?label )
-	    }
+      		?currentDescription rdaa:P50291 ?surname  .
+      		OPTIONAL {
+    			?currentDescription rdaa:P50292 ?givenName .
+			}
+      		OPTIONAL {
+        		?currentDescription foaf:title ?title .
+      		}
+      BIND(COALESCE(CONCAT(?title," ",?givenName," ",?surname),CONCAT(?givenName," ",?surname),?surname) as ?label )
+	   }
```

Comment:
This would add the `title` to the label when both `givename` and `surname` are available. Could the`title` be added to the label when only a `givename` or `surname` is present?

Reply:
Updated.